src/services/compare-appian/handlers/sendDoesNotExistAlert.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  sendAlert,
+  doesSecretExist,
+  getSecretsValue,
+  putLogsEvent,
+  trackError,
+} from "../../../libs";
+
+//! This work/logic will be done in another ticket
+
+exports.handler = async function (
+  event: { Payload: any },
+  _context: any,
+  callback: Function
+) {
+  console.log("Received event:", JSON.stringify(event, null, 2));
+
+  const region = process.env.region;
+  const project = process.env.project;
+  const stage = process.env.stage;
+
+  if (!region) throw "process.env.region needs to be defined.";
+
+  // use this secret path to define the { emailRecipients, sourceEmail } for the does not exist email
+  const secretId = `${project}/${stage}/alerts`;
```

Comment:
we may want to make these secrets appian specific. i know wale has adjusted the name for mmdl to be mmdl-alerts so maybe something like `const secretId = `${project}/${stage}/appian-alerts` then we dont have to worry about classes across features.